CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 **BREAKING CHANGES**
```

Comment:
please remove this, until we know we need it.

Reply:
done